go-controller/pkg/ovn/namespace.go

```diff
@@ -21,6 +21,8 @@ const (
 	routingExternalGWsAnnotation = "k8s.ovn.org/routing-external-gws"
 	routingNamespaceAnnotation   = "k8s.ovn.org/routing-namespaces"
 	routingNetworkAnnotation     = "k8s.ovn.org/routing-network"
+	// Enable Deny ACL Logging to log file annotation
+	aclDenyLoggingAnnotation = "k8s.ovn.org/acl-deny-logging"
```

Comment:
Thank you for the reviews! Yes, we do have a use case of adjusting it per namespace. We could switch to k8s.ovn.org/acl-logging as a map[string]string{} with deny and allow keys be separately configurable. I.e. something like this:

```
kind: Namespace
apiVersion: v1
metadata:
  name: tenantA
  annotations:
    k8s.ovn.org/acl-logging: { "deny": "alert", "allow": "notice" }
```

If "deny" or "allow" keys are defined or not defined, it will accordingly enable/disable logging logic. Thoughts?

Reply:
Out of curiosity, what's the use case for changing ACL log levels on a per namespace basis? Also using the map as the value here seems to much better idea and will help avoid the addition of extra config values, I like it

Reply:
we collecting all the logs and do post analysis, yes, filtering by logging level is a useful feature